server/core/src/server/storage.ts

```diff
@@ -879,7 +877,11 @@ export class TServerStorage implements ServerStorage {
       if (!this.hierarchy.isDerived(tx._class, core.class.TxApplyIf)) {
         if (tx.space !== core.space.DerivedTx) {
           if (this.hierarchy.isDerived(tx._class, core.class.TxCUD)) {
-            if (this.hierarchy.findDomain((tx as TxCUD<Doc>).objectClass) !== DOMAIN_TRANSIENT) {
+            const objectClass = (tx as TxCUD<Doc>).objectClass
+            if (
+              objectClass !== core.class.BenchmarkDoc &&
```

Comment:
should check DOMAIN_BENCHMARK instead?

Reply:
It is a bit faster for benchmark purpose :-)